src/harness/fake_sas.py

```diff
@@ -192,6 +192,9 @@ def GetEscSensorRecord(self, request, ssl_cert=None, ssl_key=None):
       # Return Empty if invalid Id
       return {}
 
+  def GetFullActivityDump(self):
+    return {'files':[{'url':None, 'checksum':None, 'size':0, 'version':None, 'recordType':None}]}
```

Comment:
An addition of whitespace after each key and breaking this line to have 80 characters in each line?